crates/sui-source-validation-service/src/lib.rs

```diff
@@ -148,7 +187,10 @@ pub struct CloneCommand {
 impl CloneCommand {
     pub fn new(p: &RepositorySource, dest: &Path) -> anyhow::Result<CloneCommand> {
         let repo_name = repo_name_from_url(&p.repository)?;
-        let dest = dest.join(repo_name).into_os_string();
+        let dest = dest
+            .join(p.network.as_ref().unwrap_or(&Network::Mainnet).to_string())
```

Comment:
It seems like `Network::Mainnet` is a candidate for `<Network as Default>::default()` .  You can derive `Default` (it it uses the first enum variant), or implement it explicitly, and then use `unwrap_or_default()` here and in other places, to keep source of truth for what the default network is in one place.

Reply:
I like that, I think I briefly remembered the `Default` trait and then got distracted fixing other things. Done!